app/controllers/annotations_controller.rb

```diff
@@ -352,6 +369,14 @@ def destroy
       p 'did not find @annotation for destroy: ' + params['id']
       format.json { render json: nil, status: :ok }
     else
+      puts @annotation.inspect
+      if checkAuthorization(request.headers['Authorization'],  @annotationIn["on"].first["full"]) == false
```

Comment:
@annotationIn is undefined

Reply:
When I run destroy from Drupal, I get a 404 not found for the path. Will look into.

Reply:
In my comment I meant you cannot use `@annotationIn` because it is not defined anywhere in the method. 404 is another problem: remember we found that annotation id was prefixed with duplicate host names, something like "http://example.orghttp://example.org/annotation/1"?

Reply:
Is the duplicate hostname a Mirador issue? I removed the @annotationIn and will use the Annotation object.